binderhub/tests/test_registry.py

```diff
@@ -230,7 +281,7 @@ async def test_get_image_manifest(tmpdir, token_url_known):
             f,
         )
     if token_url_known:
-        token_url = url + "/token"
+        token_url = url + "/token?service=container_registry"
```

Comment:
Is this needed? If it's testing another scenario can you use `@pytest.mark.parametrize` to test with and without this extra parameter?